### PR TITLE
Added types for 'css-to-style'

### PR DESCRIPTION
**Pull request by bengry**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [X] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [X] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [X] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [X] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by bengry** [comment]
Trying to run `npm test` results in the following error:
```console
> definitely-typed@0.0.2 test /Users/bengr/Projects/personal/DefinitelyTyped
> node node_modules/types-publisher/bin/tester/test.js --run-from-definitely-typed

Clean data
Clean logs
Clean output
# Typing Publish Report Summary
Started at Fri, 13 Jul 2018 09:52:30 GMT
Found 4652 typings folders in ~/projects/DefinitelyTyped/types
Error: amqp-connection-manager depends on amqplib but has a lower required TypeScript version.
    at checkTypeScriptVersions (~/projects/DefinitelyTyped/node_modules/types-publisher/src/check-parse-results.ts:75:11)
    at Object.<anonymous> (~/projects/DefinitelyTyped/node_modules/types-publisher/src/check-parse-results.ts:19:2)
    at Generator.next (<anonymous>)
    at fulfilled (~/projects/DefinitelyTyped/node_modules/types-publisher/bin/check-parse-results.js:4:58)
npm ERR! Test failed.  See above for more details.
```

`npm run lint css-to-style` works fine.

Please advise.

**Review by plantain-00 — APPROVED** [review]

**Comment by bengry** [comment]
@plantain-00 Do I need another approval before being able to merge this PR? I still can't merge it after your approval.

**Comment by plantain-00** [comment]
You can't merge, others who can do it, will review this and merge this.

**Pull request merged**
